docs/user/TUTORIAL.md

```diff
@@ -555,3 +555,161 @@ assert response.ok
 ```
 
 And that's it! Now we know all that we need to know to use the OSIDB REST API.
+
+
+## Fetching flaw comments
+
+### Fetching flaw comments for a specific flaw
+
+> Note: in this section we will use a made up uuid, you should use a uuid from the dataset you retrieved in the previous sections, otherwise the following examples will not work.
+
+With cURL
+
+```bash
+$ curl -H "Authorization: Bearer <access_token>" \
+       ${SERVICE_URL}/osidb/api/v1/flaws/2fe16efb-11cb-4cd2-b31b-d769ba821073/comments
+```
+
+With python
+
+```python
+import requests
+
+flaw_id = "2fe16efb-11cb-4cd2-b31b-d769ba821073"
+headers = {"Authorization": "Bearer <access_token>"}
+response = requests.get(f"${SERVICE_URL}/osidb/api/v1/flaws/{flaw_id}/comments", headers=headers)
```

Comment:
Redundant `$` (also on lines 627, 650, 674, 709).

Btw. this readme file is outdated and contains more known errors/inaccuracies (they will be fixed sometimes).